### PR TITLE
make revision a positional argument to release

### PR DESCRIPTION
**Pull request by dstathis**
fixes: #127

### CODE REVIEW

**Review by facundobatista — CHANGES_REQUESTED** [review]
This is not just a matter of making the parameter mandatory. You also need to remove the code that deals for this arg not being present, and adjust tests.

Also, add a comment to remove the corresponding case in https://docs.google.com/document/d/1Pe-0ED6db53SmrUGIAgVMzxeCOGJQwsZJWf7jzFSagQ/

And also fix `completion.bash` file

File: charmcraft/commands/store/__init__.py [hunk 1]
```diff
@@ -330,13 +330,13 @@ class ReleaseCommand(BaseCommand):
 
     def fill_parser(self, parser):
         """Add own parameters to the general parser."""
+        parser.add_argument(
+            'revision', type=int, help='The revision to release.')
         parser.add_argument(
             'channels', metavar='channel', nargs='+',
+
```
Comment (facundobatista) [inline charmcraft/commands/store/__init__.py:337]: Please remove this empty line.

**Review by facundobatista — CHANGES_REQUESTED** [review]
Almost done, great! Please check the inline comment, and remember to fix the `completion.bash` file.

File: charmcraft/commands/store/__init__.py (hunk 1 above)
Comment (dstathis) [reply charmcraft/commands/store/__init__.py]: Done.

**Review by facundobatista — APPROVED** [review]
Great, thanks! FTR, I already updated the doc after your suggestion there.

**Review by chipaca — APPROVED** [review]

**Pull request merged**